tests/integration_test.py

```diff
@@ -598,232 +603,347 @@ def test_get_phenomena_by_name():
+        self.assertIsInstance(res, Success, res.__str__())
+        self.assertTrue(len(res.result) > 0)
+        for subscription in res.result:
+            self.assertIsInstance(subscription, Subscription)
             print(subscription.code, 'from', subscription.owner, 'in', subscription.link)
 
         print('\n')
 
-    @staticmethod
-    def test_around():
+    def test_subscriptions_by_active_status(self):
+        print('Testing Subscriptions ========================')
+
         # given
-        user = 'ensoag'
-        subscription = '3JSM4MN89SJFLE7VR6KPCM0BVPXTWT'
-        session = _get_sandbox_session()
-        query = Query().subsc.by_user(user).filter_by(subscription).around(5, '20200105', '20200115')
+        session = _get_dev_session()
+        query = Query().subsc.by_status(1)
 
         # when
-        entity = Context(session).get(query)
+        res = Context(session).get(query)
 
         # then
-        assert isinstance(entity, BinaryData)
-        assert subscription in entity.name
-        assert len(entity.content) > 0
+        self.assertIsInstance(res, Success, res.__str__())
+        self.assertTrue(len(res.result) > 0)
+        for subscription in res.result:
+            self.assertIsInstance(subscription, Subscription)
+            self.assertEqual(1, subscription.status)
+            print(subscription.code, 'from', subscription.owner, 'in', subscription.link)
+
+        print('\n')
+
+    def test_subscriptions_by_user_with_active_status(self):
+        print('Testing Subscriptions ========================')
 
-    @staticmethod
-    def test_last():
         # given
-        user = 'ensoag'
-        subscription = '3JSM4MN89SJFLE7VR6KPCM0BVPXTWT'
-        session = _get_sandbox_session()
-        query = Query().subsc.by_user(user).filter_by(subscription).last
+        session = _get_dev_session()
+        query = Query().subsc.by_user(self._user).by_status(1)
 
         # when
-        entity = Context(session).get(query)
+        res = Context(session).get(query)
 
         # then
-        assert isinstance(entity, BinaryData)
-        assert subscription in entity.name
-        assert len(entity.content) > 0
+        self.assertIsInstance(res, Success, res.__str__())
+        for subscription in res.result:
+            self.assertIsInstance(subscription, Subscription)
+            self.assertEqual(1, subscription.status)
+            print(subscription.code, 'from', subscription.owner, 'in', subscription.link)
 
-    @staticmethod
-    def test_last_n():
+        print('\n')
+
+    def test_around(self):
         # given
-        user = 'ensoag'
-        subscription = '3JSM4MN89SJFLE7VR6KPCM0BVPXTWT'
-        session = _get_sandbox_session()
-        query = Query().subsc.by_user(user).filter_by(subscription).last_n(5)
+        session = _get_dev_session()
+        query = Query().subsc.by_user(self._user).filter_by(self._subscription).results.around(5, '20200519', '20200622')

```

Comment:
change "**filter_by**" to "**by_key**"  ??

Reply:
Addressed